tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+                "VK_AMD_extension_134",
+                "[DO NOT EXPOSE] We don't support yet",
+                "VK_AMDX_shader_enqueue",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_136",
+                "[TODO] need impl",
+                "VK_AMD_mixed_attachment_samples",
+                "OR",
+                "VK_NV_framebuffer_mixed_samples"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "mixedAttachmentSamples",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "[TODO LATER] Wait until VK_AMD_shader_fragment_mask & VK_QCOM_render_pass_shader_resolve converge to something useful",
+                "VK_AMD_shader_fragment_mask",
+                "[DEPRECATED] Required wholly by ROADMAP 2022 and Nabla Core Profile",
+                "InlineUniformBlockFeaturesEXT",
+                "VK_EXT_inline_uniform_block",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_140",
+                "Enabled by Default, Moved to Limits",
+                "VK_EXT_shader_stencil_export",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_142",
+                "VK_AMD_extension_143",
+                "[DO NOT EXPOSE] We don't support yet",
+                "VK_EXT_sample_locations",
+                "[DEPRECATED] Promoted to non-optional core VK 1.1",
+                "VK_KHR_relaxed_block_layout",
+                "[DO NOT EXPOSE] We don't support yet",
+                "VK_RESERVED_do_not_use_146",
+                "[DEPRECATED] Promoted to core non-optional VK 1.1",
+                "VK_KHR_get_memory_requirements2",
+                "[DEPRECATED] Vulkan 1.2 core non-optional",
+                "VK_KHR_image_format_list",
+                "[DO NOT EXPOSE] This is dumb, you can implement whatever blend equation you want with `EXT_fragment_shader_interlock` and EXT_shader_tile_image",
+                "BlendOperationAdvancedFeaturesEXT",
+                "VK_EXT_blend_operation_advanced",
+                "[TODO LATER] Requires API changes",
+                "VK_NV_fragment_coverage_to_color",
+                "AccelerationStructureFeaturesKHR",
+                "VK_KHR_acceleration_structure"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "accelerationStructure",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "accelerationStructureIndirectBuild",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "accelerationStructureHostCommands",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE] implied by `accelerationStructure`"
+            ]
+        },
```

Comment:
thats not a new group, just a comment for `descriptorBindingAccelerationStructureUpdateAfterBind`